tests/isin_test.py

```diff
@@ -17,3 +20,15 @@ def test_isin():
     assert df.w.isin([2, None]) == [True, False, True]
     assert df.m.isin([1, 2, 3]) == [False, False, True]
     assert df.n.isin([2, np.nan]) == [False, True, False]
+
+
+def test_isin_pandas_object():
+    df_pandas = pd.DataFrame({'x': np.array(['a', 'b', 'c']),
+                              'y': np.array([1, 2, 3], dtype='O')})
```

Comment:
I think it has nothing to do with pandas, but with dtype=object? If so, could you rename everything? Now it is misleading as if we treat data from pandas differently. I understand that it shows up when using a pandas dataframe, but it is no the underlying issue.